### ISSUE TITLE
Sometimes shows nothing in the Store

### ISSUE THREAD

**Issue by samuela** [relayjs/relay-devtools#95]
Sometime when I open up Relay devtools (1.0.0-096c4246b84d+) I get a cool view into everything that's in the store. But sometimes it's just totally empty except for `__Root > client:root` :/

**Comment by mmmols** [comment]
Yeah, it's a known bug. Refresh your window it it should work again.

**Comment by thibaultboursier** [comment]
Is there a way to fix this bug?
